It is confusing to get lots of "efi_runtime: can't get variable" error messages during test.
It will be better for the caller to determine the error message against efi return status.
---
 efi_runtime/efi_runtime.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/efi_runtime/efi_runtime.c b/efi_runtime/efi_runtime.c
index 31e5bb3..7c14b23 100644
--- a/efi_runtime/efi_runtime.c
+++ b/efi_runtime/efi_runtime.c
@@ -288,7 +288,6 @@ static long efi_runtime_get_variable(unsigned long arg)
 			return -EFAULT;
 		return 0;
 	} else {
-		printk(KERN_ERR "efi_runtime: can't get variable\n");
 		return -EINVAL;
 	}
 
-- 
2.1.4
